gallery/src/styles/NxPageLocalHeader/NxPageLocalHeaderEmptyExample.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {
+  NxGlobalSidebarNavigation,
+  NxGlobalSidebarNavigationLink,
+  NxStatefulGlobalSidebar,
+  NxPageMain
+} from '@sonatype/react-shared-components';
+import { faArrowLeft, faArrowRight, faLink } from '@fortawesome/free-solid-svg-icons';
+
+const sidebarLogoPath = require('../../assets/logo-plaid-villain-text.png');
+
+export default function NxPageLocalHeaderEmptyExample() {
+  return (
+    <>
+      <aside className="nx-page-local-header"></aside>
```

Comment:
Why "local"? Since this is meant to be used with the "global" sidebar I'd have expected a classname that would have connected the two in a clearer way.

Reply:
> Why "local"? Since this is meant to be used with the "global" sidebar I'd have expected a classname that would have connected the two in a clearer way.

It was a difficult choice.  The problem is not only how to link it to the global sidebar, but how to differentiate it from the old page header.  Really, in the old designs the page header was global while the sidebar was local to the page.  In the new designs the situation is reversed.  So it really seemed wrong to call this header the "global" one when it is in pretty much every way less global than the old header.  It also contains some page-specific content in the form of the back button.  I'm open to suggestions but I really don't think anything mentioning "global" is right for this.

Reply:
The more I think about the more I dislike "local" here. I don't think that there's any intent that this header would be used with anything but `NxGlobalSidebar` so I think that they should be tied intrinsically together.

Reply:
> I'm open to suggestions but I really don't think anything mentioning "global" is right for this.

I'll think about it. In retrospect "Global" was a shitty name choice. Sometimes I think that we should have just numbered the damn things. Sidebar1 goes with Header1, Sidebar2 goes with Header2, etc.

Reply:
> less global than the old header

Is this true though? The buttons to the right are always present aren't they?

Reply:
> The more I think about the more I dislike "local" here. I don't think that there's any intent that this header would be used with anything but NxGlobalSidebar so I think that they should be tied intrinsically together.

Renamed: 866f7b7

Reply:
>  I think that we should have just numbered the damn things. Sidebar1 goes with Header1, Sidebar2 goes with Header2, etc.

+1!